### DISCUSSION TITLE
Support for Spring 6

### DISCUSSION THREAD

**Discussion by SebastianOltmanns** [flamingock/mongock#29] [General]
Spring 6 / Spring Boot 3 was released in November last year. When will Mongock support the new versions?

**Comment by osantana85** [comment]
Hello @SebastianOltmanns , Mongock already supports Springboot v3 (and springdata v4). This [sample project](https://github.com/mongock/mongock-examples/tree/master/mongodb/springboot3-springdata4-jdk17) shows you how to use it (JDK17).

Basically you need to import a new runner + driver, this way:

```       
        <!-- MONGOCK RUNNER -->
        <dependency>
            <groupId>io.mongock</groupId>
            <artifactId>mongock-springboot-v3</artifactId>
        </dependency>

        <!-- MONGOCK DRIVER -->
        <dependency>
            <groupId>io.mongock</groupId>
            <artifactId>mongodb-springdata-v4-driver</artifactId>
        </dependency>
```

Hope this helps you.

Thanks for using Mongock!

**Reply by SebastianOltmanns** [reply]
Thank you @osantana85 , it worked.

Please note that I just read the documentation and it says nothing about spring data v4. That's why I assumed there was no support yet. I suggest updating the documentation: https://docs.mongock.io/v5/driver/mongodb-springdata/index.html

**Reply by osantana85** [reply]
@SebastianOltmanns you are right, we’ll update the documentation soon. Thanks.
